### ISSUE TITLE
Cooperation with solaire-mode

### ISSUE THREAD

**Issue by WhatDothLife** [ema2159/centaur-tabs#141]
> solaire-mode is an aesthetic plugin that helps visually distinguish file-visiting windows from other types of windows (like popups 
> or sidebars) by giving them a slightly different -- often brighter -- background.

I'd love to see some cooperation with hlissner's solaire mode. So far the buffers brightened by solaire-mode (like the scratch buffer) look like this:
![image](https://user-images.githubusercontent.com/33008303/101658307-2d2cd780-3a45-11eb-8a51-875026c7156a.png)

Maybe, centaur-tabs could offer a function for swapping all backgrounds of the tabbar, that can be run as a hook. Let me know what you think of it.

**Comment by ema2159** [comment]
Indeed. This has been one of the features I've been wanting to implement the most since quite some time ago. The problem is that the way Emacs face remapping works makes the task quite difficult so at some point I stopped trying, but I may try again. If you find some solution though, let me know as soon as possible!

**Comment by WhatDothLife** [comment]
Ok, thanks for the update.
